### ISSUE TITLE
Pre-processing from training set applied to test set

### ISSUE THREAD

**Issue by benkraj** [svkucheryavski/mdatools#40]
Hi Sergey-

I was wondering if when I apply a pre-processing step (i.e. center=TRUE)  with pls() or ipls() if this is then applied to test sets when I use predict()? If so, do I need to pass any other arguments in predict i.e. method='center' or something else?

I know this is the functionality in a few other packages---and was curious if this is the right approach here as well.

Thanks for any help.
-Ben

**Comment by svkucheryavski** [comment]
Hi Ben,

Yes, model saves mean values and standard deviations (if you use scaling) computed for calibration set and when you apply this model to a new object or dataset it applies these values for centering and scaling. The same for PCA and SIMCA. So you do not need to pass any other arguments.

Feel free to write me if you have any other questions.

Sergey

**Comment by benkraj** [comment]
Great. Thanks for the clarification Sergey, and thanks for the useful package!
